### PR TITLE
26 develop initial interface

### PR DESCRIPTION
**Pull request by sergiomrebelo**
A first version of the initial interface is created. It still necessary to improve code structure (see issue https://github.com/sergiomrebelo/evo-poster/issues/34) and evaluation (see issue https://github.com/sergiomrebelo/evo-poster/issues/36).

### CODE REVIEW

**Comment by JJ** [comment]
If you really want me to comment on them, you should try and leave them open for a bit longer...

File: cypress/e2e/generator.cy.js [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+            cy.get('#random-colour-typo-check').should('be.visible').should('be.checked');
+            cy.get('#random-colour-typo-check').uncheck();
+            cy.get('#typography-colour-picker-1').should('not.be.disabled');
+
+            cy.get('#typography-colour-picker-1').invoke('val', inputText[lang]["divider"]["typography"]["mainColor"]["testingColor"]).trigger('change');
+            cy.get('#typography-colour-picker-1').scrollIntoView();
+
+            // verify slider weight
+            // min
+            cy.get('#typeface-weight-min-slider').should('exist');
```
Comment (JJ) [inline cypress/e2e/generator.cy.js:150]: You can possibly extract common code and run a loop over a list of what you want to do.

**Review by JJ — COMMENTED** [review]
I'm not sure it's a good idea to include fonts in the repository. They load the repo a lot, can't really be reviewed, and they can easily be lazily loaded from th eweb. You need to consider the license, too, and include it in the repository.

File: cypress/e2e/generator.cy.js (hunk 1 above)
Comment (sergiomrebelo) [reply cypress/e2e/generator.cy.js:150]: Yeah. It makes total sense (assign a specific class to testing elements and run all at the same time). I will do it in the next test script.

**Comment by sergiomrebelo** [comment]
> I'm not sure it's a good idea to include fonts in the repository. They load the repo a lot, can't really be reviewed, and they can easily be lazily loaded from th eweb. You need to consider the license, too, and include it in the repository.

The typefaces that I selected are open source and available freely on the web. Do you know if there is a way of automatically downloading the files, from the production folder? I think that this can be a nice solution. Anyway, people can download it from the web easily. I can also remove them.

**Comment by JJ** [comment]
> > I'm not sure it's a good idea to include fonts in the repository. They load the repo a lot, can't really be reviewed, and they can easily be lazily loaded from th eweb. You need to consider the license, too, and include it in the repository.
> 
> The typefaces that I selected are open source and available freely on the web. Do you know if there is a way of automatically downloading the files, from the production folder? I think that this can be a nice solution. Anyway, people can download it from the web easily. I can also remove them.

You can probably use rollup for that. But I don't think it's worth the while to do either. It's going to give you a marginal speed gain, since they will be served from your disk, and only if you do it locally. Google CDNs are going to be faster that whatever you use for hosting in production.

**Pull request merged**